aggregator_api/src/routes.rs

```diff
@@ -76,19 +76,16 @@ pub(super) async fn get_task_ids<C: Clock>(
         })?;
     let pagination_token = task_ids.last().cloned();
 
-    Ok((
-        Json(GetTaskIdsResp {
-            task_ids,
-            pagination_token,
-        }),
-        Halt,
```

Comment:
Hmm, I think maybe we had to do this in order to stop other routes from also matching on the request, but I don't think any other handler would have matches on `/task_ids` so I suspect this wasn't needed.

Reply:
Looking into it more I think it'll stop any additional handlers from running, but we don't have any at the moment. https://github.com/divviup/janus/blob/8c2761d4c438bf488227a90d66c2682b8bbb2d7b/aggregator_api/src/lib.rs#L71